src/middleware/authentication/middleware.ts

```diff
@@ -1,18 +1,37 @@
+// biome-ignore lint/correctness/noNodejsModules: we need the node crypto module
+import { createHash } from "node:crypto";
 import type { NextFunction, Request, Response } from "express";
 import { introspectToken } from "@lib/idp/introspectToken";
+import { RedisConnector } from "@src/lib/redisConnector";
+
+const cacheExpiry = 60; // seconds
+
+export function getAccessTokenCacheKey(accessToken: string): string {
+  const hash = createHash("sha256").update(accessToken).digest("hex");
```

Comment:
A base64 digest can shorten the resulting string by almost 45%
An example in CodeSandbox with a real access token:
```text
Hash with Hex length 64: 28cfceba191f40187af806b5b18ecca20ecb94366a1712bfe840bc26a8ca6204
Has with base64 length 44: KM/OuhkfQBh6+Aa1sY7Mog7LlDZqFxK/6EC8JqjKYgQ=
String shortened by 45.45454545454546%
```

```suggestion
  const hash = createHash("sha256").update(accessToken).digest("base64");
```

Reply:
Nice, I'll get that change in